include/pmacc/eventSystem/tasks/ITask.hpp

```diff
@@ -49,16 +49,21 @@ namespace pmacc
             TASK_HOST
         };
 
+        /** Flops, Bytes, and timer for this task.
+         */
+        PerfData perfInfo;
+
         /**
          * constructor
          */
-        ITask() : myType(ITask::TASK_UNKNOWN)
+        ITask() : myType(ITask::TASK_UNKNOWN), perfInfo(0, 0)
```

Comment:
@j-stephan this is sexy to measure the lifetime of an object, that's maybe something for bactria too.